packages/massa-web3/src/web3/SmartContractsClient.ts

```diff
@@ -338,29 +347,39 @@ export class SmartContractsClient
    * @returns A promise that resolves to the status of the operation.
    */
   public async getOperationStatus(opId: string): Promise<EOperationStatus> {
-    const operationData: Array<IOperationData> =
-      await this.publicApiClient.getOperations([opId])
-
-    if (!operationData?.length) return EOperationStatus.NOT_FOUND
+    const [operation] = await this.publicApiClient.getOperations([opId])
```

Comment:
this will throw an error if `await this.publicApiClient.getOperations([opId])` return an empty array

Reply:
Thanks!